### PR TITLE
added docker

### PR DESCRIPTION
**Pull request by BlueVioletTeti**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Builder stage
+FROM openjdk:17-jdk-alpine as builder
+WORKDIR application
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+#Final stage
```
Comment (Denis-Balako) [inline Dockerfile]: Remove unnecessary comments

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/books_app?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=imdv306
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql://localhost:3306/books_app?serverTimezone=UTC
+#spring.datasource.username=root
+#spring.datasource.password=imdv306
+#spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
```
Comment (Denis-Balako) [inline src/main/resources/application.properties]: Remove unnecessary comments

File: src/test/java/com/vozniuk/bookstore/BookstoreApplicationTests.java
Comment (Denis-Balako) [inline src/test/java/com/vozniuk/bookstore/BookstoreApplicationTests.java:1]: Remove unnecessary comments

**Review by Denis-Balako — APPROVED** [review]
Great! Left few optional suggestions.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,7 +1,7 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/books_app?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=imdv306
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+#spring.datasource.url=jdbc:mysql://localhost:3306/books_app?serverTimezone=UTC
```
Comment (andrii-hoienko) [inline src/main/resources/application.properties]: Is it possible to remove unnecessary comments?

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
Please remove unnecessary comments

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**